src/main/java/com/crowdin/cli/commands/picocli/CommandNames.java

```diff
@@ -56,4 +56,7 @@ public final class CommandNames {
     public static final String TASK = "task";
     public static final String TASK_LIST = "list";
     public static final String TASK_ADD = "add";
+    public static final String BUNDLE = "bundle";
+    public static final String BUNDLE_LIST = "list";
+    public static final String BUNDLE_ADD = "add";
```

Comment:
the download is missing?

Reply:
we already have download command implemented:
crowdin download bundle

so I just added download bundle subcommand